ansible/roles/screenly/tasks/main.yml

```diff
@@ -30,6 +30,13 @@
   ansible.builtin.pip:
     requirements: "/home/{{ lookup('env', 'USER') }}/screenly/requirements/requirements.host.txt"
     extra_args: "--no-cache-dir --upgrade"
+  when: ansible_distribution_major_version|int <= 11
+
+- name: Install pip dependencies
+  ansible.builtin.pip:
+    requirements: "/home/{{ lookup('env', 'USER') }}/screenly/requirements/requirements.host.txt"
+    extra_args: "--no-cache-dir --upgrade --break-system-packages"
```

Comment:
That looks a bit scary.

Reply:
If it's about the `--break-system-packages`, I could check if the installer will still do the same thing if I use a virtual environment. The new `pip` version enforces the use of a virtual environment when installing Python packages globally.

Reply:
I mean that is sensible. We should probably adopt a venv rather than installing things globally.